libs/sql-schema-describer/src/walkers.rs

```diff
@@ -407,14 +380,14 @@ pub trait SqlSchemaExt {
 impl SqlSchemaExt for SqlSchema {
     fn table_walker<'a>(&'a self, name: &str) -> Option<TableWalker<'a>> {
```

Comment:
Could this be rewritten as

```suggestion
    fn table_walker(&self, name: &str) -> Option<TableWalker<'_>> {
```
?

Reply:
I suspect not (see previous comment), but I can try.